src/server/templates/page.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function page({ body = "", assets, title }) {
+  return `
```

Comment:
For the moment it is just to get going. You are right, if we need to have more server side templates then we should definitely swap to Handlebars. 👍

I'm assuming the whole page template will be the only one we will need. The rest of the rendering of views should be happening withing preact or whatever.